src/sentry/utils/numbers.py

```diff
@@ -62,3 +62,17 @@ def base36_encode(number):
 
 def base36_decode(s):
     return int(s, 36)
+
+
+def format_bytes(number, units=['B', 'KB', 'MB', 'GB', 'TB',
+                                'PB', 'EB', 'ZB', 'YB'], decimal_places=2):
```

Comment:
Will units ever be used as argument? You have a mutable type as a default arg which doesn't work great.

Reply:
micro ptimization -- it only matters if you were gonna mutate it, but ill fix, and yes it'll be used